packages/edit-site/src/components/sidebar-navigation-screen/style.scss

```diff
@@ -37,10 +37,6 @@
 }
 
 .edit-site-sidebar-navigation-screen__title {
-	font-size: calc(1.56 * 13px);
```

Comment:
Heh, I know it's subjective but the 20px feels kind of oversized to me, particularly with the longer titles.

I like 16 because it aligns nicely with the icons:

<img width="325" alt="Screenshot 2023-03-31 at 15 17 59" src="https://user-images.githubusercontent.com/846565/229146114-250e4bb2-0f50-434e-aa42-97938f2f3652.png">

It's also what we've been circling around with mockups like:

<img width="346" alt="Screenshot 2023-03-31 at 15 20 18" src="https://user-images.githubusercontent.com/846565/229146892-95b4575f-5061-4e36-9ccd-0197ea9378d4.png">
<img width="349" alt="Screenshot 2023-03-31 at 15 20 53" src="https://user-images.githubusercontent.com/846565/229146897-cf4e9fd7-560f-4325-bf3d-945669a316e2.png">

This is only a light pushback though, I don't mind working on that separately :)

Reply:
6px is a grid unit! It's `$grid-unit-15 * 0.5` ;) 

It's not a change I'd fight tooth and nail against. But mainly I think it's important we have some contrast between the various headings, just so we don't end up with a bunch of font sizes that are close but not quite contrasty enough. At the moment we have 11px labels (allcaps), 12px help text, 13px body size for the small stuff. Then we have a few sizes above here that don't seem entirely systematized, something around 14px for modal subheadings, and 19ish for modal headers. It's a bit messy and could use a system in our library. 16 may be a part of that, but then I'd roll it out more broadly. Just visually it doesn't seem far enough from 13px.

If you want the 16px change in this PR, I'm happy to make @richtabor or @SaxonF a tie-breaker :D

Reply:
> 6px is a grid unit! It's $grid-unit-15 * 0.5 ;)

😆

Oh absolutely, we should define a scale, ideally built into the `Text` / `Heading` components. We have [this](https://www.figma.com/file/e4tLacmlPuZV47l7901FEs/WordPress-Design-Library?node-id=28593%3A27082&t=YBQ7fPCKSbDdVskG-1) in the Design Library, but like you said only the only ones close to being formalised in any way are Label, Helper text, and Body. We should revisit that now we're beginning to explore management screens in some [more detail](https://github.com/WordPress/gutenberg/compare/try/pages-list).

No need for a tie-breaker haha, I reverted.

Reply:
I don't mind the current state's 20px size.